ocs_ci/ocs/utils.py

```diff
@@ -831,12 +831,11 @@ def collect_ocs_logs(dir_name, ocp=True, ocs=True, mcg=False):
         )
     if mcg:
         counter = 0
-        mcg_db_collected = False
-        while not mcg_db_collected or counter < 5:
+        while counter < 5:
```

Comment:
Why not use our [retry decorator](https://github.com/red-hat-storage/ocs-ci/blob/master/ocs_ci/utility/retry.py) for `collect_noobaa_db_dump`?

Reply:
This was done by @ebenahar , and I am just fixing the issue here. The retry will anyway the re-raise the exception after final retry. This is not the case here as we are just ignoring it here and continue without re-raising the exception.